Source/UnrealEnginePython/Private/UnrealEnginePython.cpp

```diff
@@ -113,7 +113,12 @@ void FUnrealEnginePythonModule::RunString(char *str) {
 
 void FUnrealEnginePythonModule::RunFile(char *filename) {
 	FScopePythonGIL gil;
-	char *full_path = TCHAR_TO_UTF8(*FPaths::Combine(*FPaths::GameContentDir(), UTF8_TO_TCHAR("Scripts"), *FString("/"), UTF8_TO_TCHAR(filename)));
+	char *full_path = (char *)malloc(strlen(filename) + 1);
```

Comment:
Sounds good, will update this shortly. I wasn't sure if there would be issues changing filename in the case of filename not existing.